test/cppsuite/tests/test_template.cpp

```diff
@@ -28,60 +28,84 @@
 
 #include "test_harness/test.h"
 
+/* Class that specifies what to write into the tracking table. */
+class tracking_table_template : public test_harness::workload_tracking {
+
+    using workload_tracking::workload_tracking;
+
+    int
+    populate_tracking_cursor(const tracking_operation &operation, const uint64_t &collection_id,
+      const char *key, const char *value, wt_timestamp_t ts,
+      scoped_cursor &op_track_cursor) override final
+    {
+        /* You can replace this call to define your own tracking table contents. */
+        workload_tracking::populate_tracking_cursor(
+          operation, collection_id, key, value, ts, op_track_cursor);
+    }
+};
+
 /*
  * Class that defines operations that do nothing as an example. This shows how database operations
  * can be overridden and customized.
  */
 class test_template : public test_harness::test {
     public:
-    test_template(const test_harness::test_args &args) : test(args) {}
+    test_template(const test_harness::test_args &args) : test(args)
+    {
+        delete this->_workload_tracking;
+        this->_workload_tracking =
+          new tracking_table_template(_config->get_subconfig(WORKLOAD_TRACKING),
+            _config->get_bool(COMPRESSION_ENABLED), *_timestamp_manager);
+    }
 
     void
     run() override final
     {
-        /* You can remove the call to the base class to fully customized your test. */
+        /* You can remove the call to the base class to fully customize your test. */
         test::run();
     }
 
     void
-    populate(test_harness::database &, test_harness::timestamp_manager *,
-      test_harness::configuration *, test_harness::workload_tracking *) override final
+    populate(test_harness::database &database, test_harness::timestamp_manager *tsm,
```

Comment:
The arguments were left without a name on purpose here. When an arg is not used in a function, it is fine to leave it without a name, can you revert that commit?

Reply:
Reverted in e508f6a612fa3032d303ee01638dfa4a55030a69